rust/worker/src/execution/orchestration/hnsw.rs

```diff
@@ -264,10 +282,13 @@ impl HnswQueryOrchestrator {
 
         // Dispatch a query task
         let operator = Box::new(HnswKnnOperator {});
+        let disallowed_ids = self.get_disallowed_ids(logs);
         let input = HnswKnnOperatorInput {
             segment: hnsw_segment_reader,
             query: self.query_vectors[0].clone(),
             k: self.k as usize,
+            allowed_ids: self.allowed_ids.clone(),
```

Comment:
one note is that we will need to turn these ids into offset ids for hnsw

Reply:
Yes. Do you have an example for id conversion?